### PR TITLE
Fix bower install

### PR DESCRIPTION
**Pull request by blimmer**

### CODE REVIEW

File: blueprints/ember-cli-mocha/index.js [hunk 1]
```diff
@@ -12,17 +12,14 @@ module.exports = {
...
-      })
-      .then(function() {
-        return addonContext.addBowerPackageToProject('ember-cli/ember-cli-shims', '0.0.3');
-      })
-      .then(function() {
-        if ('removePackageFromProject' in addonContext) {
-          return addonContext.removePackageFromProject('ember-cli-qunit');
-        }
-      });
+    return this.addBowerPackagesToProject([
```
Comment (blimmer) [inline blueprints/ember-cli-mocha/index.js]: This was broken by [this changeset in ember-cli](https://github.com/ember-cli/ember-cli/commit/f87760ca966dfa1653e54a7c1349fd658c94c121).

**Comment by blimmer** [comment]
This is a quick fix, but @stefanpenner and I [agree](https://github.com/ember-cli/ember-cli-qunit/issues/72#issuecomment-128085631) the old way should still work. With that said, this will fix the issue for those trying to use this project with today's flavor of ember-cli.

**Comment by dgeb** [comment]
Thanks @blimmer!

Published in v0.9.2.

**Comment by stefanpenner** [comment]
this fix works, but only the source for `ember-cli-shims' needs to be specified since it is not published to bower (yet)

**Comment by stefanpenner** [comment]
If someone wants to publish shims to bower, we can drop the source here too.

**Pull request merged**